### ISSUE TITLE
Link to Reflective Journal needed ASAP!

### ISSUE THREAD

**Issue by HeatherStrong11** [TWUOnline/bug-tracking#365]
**Which course contains the bug?**  
- Course Code, section, and term (e.g. LDRS 591 OL SP2019)
 LDRS 591 A SU 2020
 
**Describe the error and/or your suggestion**  
I can't seem to link the content from the reflective learning journal from the file in github to show up in Moodle.  I used to have a link to this info on my Wordpress site, but I don't want students visiting the Wordpress site.  So how do I create a link to the reflective learning journal info (it is on a page under the assignments folder).  This link is needed by Tuesday AM!
 
**Location of the bug**  
Please provide a specific link to the page with the error, as well as where the error is on the page:
1. URL:  https://learn.twu.ca/course/view.php?id=13220&section=2
2. Unit #   1      Topic #   1      Activity#  1 - in the alert
 
**Optional: Screenshots**

**Comment by karenroeck** [comment]
@HeatherStrong11 I hyperlinked the "Introduction to the Reflective Journal" page in the Activity 1.1 alert - is this what you were hoping for?

**Comment by HeatherStrong11** [comment]
@karenroeck Yes!  Thank you.  Can you tell me how you did that?  I would also love it if there were hyperlinks from the front page of the assignments tab to each individual assignment page.  Is this possible?

**Comment by karenroeck** [comment]
@HeatherStrong11 My pleasure! I moved the reflective-journal folder into 07.1-scholary-inquiry, then I created a hyperlink to that folder in the alert in 02.topic-1:

![Screen Shot 2020-07-29 at 11 33 04 AM](https://user-images.githubusercontent.com/67760065/88839054-4ec0ad00-d18f-11ea-8713-2a282a016c5a.png)

RE: Assignment hyperlinks - the way it is set up now, each individual assignment page has its own child tab under the parent Assignments tab. Can students navigate to the individual assignment pages from here? Or is there another format you would like?
